### ISSUE TITLE
[DOCS] 8.4.0 Release notes 

### ISSUE THREAD

**Issue by nastasha-solomon** [elastic/security-docs#2175]
## Description
This issue tracks release notes for the 8.4.0 release, which is GAing on **August 23, 2022**. 

## Set up the release notes 
Before starting the release notes, do the following:
- [ ] Set up a new release notes file
- [ ] Update the release notes index file (`release-notes.asciidoc`)
- [ ] Prep the breaking changes section in the Stack docs (`breaking.asciidoc`)
- [ ] Prep the breaking changes in the Security docs (`8.4.asciidoc`)

### Set up the release notes file
Create a new release notes file for major and minor releases. To create the file, follow these steps:

1. Check out the `main` branch.
2. Run `git fetch origin` then `git pull` to pull in the latest changes.
3. Run `git -b <branch name>` to create a new branch. For example, `git -b issue-2175-8.4.0-rn`.
4. Open the Security docs in your text editor (Atom or VS Code) and navigate to the `release-notes` folder. 
5. Create a new asciidoc file in the `release-notes` folder. 
**NOTE:** Remember to include the `asciidoc` file extension when you name the file. 
6. Add a heading anchor to the top of the file. It should be `[[release-notes-header-8.4.0]]`. You'll be referencing this anchor later when you insert a cross reference to the 8.4.0 release notes into the release notes index file (`release-notes.asciidoc`).
7. Under the heading anchor, add the page title. It should be: `== 8.3`. 
8. Save the changes and open a PR. 

### Update the release notes index file
Insert a cross reference to the appropriate release notes file and add the release notes to the main TOC.

1. Open the `release-notes.asciidoc` file.  
2. Under the section that summarizes the changes in each release, add a cross reference to the 8.4.0 release notes. It should be: `* <<release-notes-8.4.0, {elastic-sec} version 8.4.0>>`. 
3. Go the end of the file and add an include for the the 8.4.0 release notes. It should be: `include::release-notes/8.4.asciidoc[]`
4. Save the changes and push them to your release notes PR. 

### Prep the breaking changes section in the Stack docs
To link the "Breaking changes" section in the Security release notes from the Elastic Installation and Upgrade guide:
1. Fork and then locally clone the [stack-docs](https://github.com/elastic/stack-docs) repo. 
2. Pull in the latest changes from the `stack-docs` repo's `main` branch. 
3. Open the `stack-docs` repo in your text editor and navigate to the `breaking.asciidoc` file.
4. Go to the "Elastic Security breaking changes" section and make the following changes:
   - Update the release version number that's included in the URL that references the latest Security release notes. For this release, the referenced URL should be: `{security-guide-all}/8.4/release-notes-header-8.4.0.html#breaking-changes-8.4.0[{elastic-sec} breaking changes]`
   -  Update the include statement so it refences the `8.4` release notes asciidoc file. For this release, ensure the include statement is: `include::{security-repo-dir}/release-notes/8.4.asciidoc[tag=breaking-changes]`
5. Save your changes and open a PR. You can model the issue and PR for updating the Stack doc's `breaking.asciidoc` file after:
   - https://github.com/elastic/stack-docs/issues/2075
   - https://github.com/elastic/stack-docs/pull/2076
   **NOTE:** The CI checks for your Stack docs PR will fail until you've merged your PR for the 8.4 Security release notes into the `main` and `8.4` branches. 

### Prep the breaking changes section in the Security docs
1. Return to your 8.4 release notes branch in the Security docs and open the 8.4 release notes file (`8.4.asciidoc`). 
2. Insert the Breaking Changes section above the Features section.  Copy and paste the following:
 
````
[discrete]
[[breaking-changes-8.4.0]]
==== Breaking changes
// tag::breaking-changes[]
// NOTE: The breaking-changes tagged regions are re-used in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
:pull: https://github.com/elastic/kibana/pull/
**TBD**
// end::breaking-changes[]
````
3. Save the changes and push them to your release notes PR.

## Troubleshooting
- Refer to the Asciidoc spec for [including content of tagged regions](https://docs.asciidoctor.org/asciidoc/latest/directives/include-tagged-regions) for more on how includes and tagged snippets work. 
- Links inside tagged content must be full URL links, not relative links or an internal anchor references. Using attributes to represent part of the URL is OK. Basically, treat links as if they're [external](https://github.com/elastic/docs#external-links) or [cross-document links](https://github.com/elastic/docs#cross-document-links).

**Comment by nastasha-solomon** [comment]
Make sure to add the following summary to the fixed issues section:
Isolates the query used for the *First seen* and *Last seen* fields on the Users, Hosts, and Network details pages to improve page load times and UI performance ({pull}133539[#133539]).

This was removed from the 8.3.3 RN because it was only merged into the 8.4 branch.

cc: @benironside
